test/fixtures/minio-fixture/build.gradle

```diff
@@ -11,26 +11,20 @@ apply plugin: 'elasticsearch.cache-test-fixtures'
 description = 'Fixture for MinIO Storage service'
 
 configurations.all {
-  transitive = false
+  exclude group: 'org.hamcrest', module: 'hamcrest-core'
 }
-
 dependencies {
-  testImplementation project(':test:framework')
-
+  testImplementation (project(':test:framework'))
   api "junit:junit:${versions.junit}"
   api project(':test:fixtures:testcontainer-utils')
-  api "org.testcontainers:testcontainers:${versions.testcontainer}"
-  implementation "com.carrotsearch.randomizedtesting:randomizedtesting-runner:${versions.randomizedrunner}"
-  implementation "org.slf4j:slf4j-api:${versions.slf4j}"
-  implementation "com.github.docker-java:docker-java-api:${versions.dockerJava}"
-
-  runtimeOnly "com.github.docker-java:docker-java-transport-zerodep:${versions.dockerJava}"
-  runtimeOnly "com.github.docker-java:docker-java-transport:${versions.dockerJava}"
-  runtimeOnly "com.github.docker-java:docker-java-core:${versions.dockerJava}"
-  runtimeOnly "org.apache.commons:commons-compress:${versions.commonsCompress}"
-  runtimeOnly "org.rnorth.duct-tape:duct-tape:${versions.ductTape}"
+  api("org.testcontainers:testcontainers:${versions.testcontainer}") {
+    transitive = false
```

Comment:
Hmm. I'm not aware of us doing this anywhere else. There's the larger question of how we want to deal with transitive dependencies going forward but should we just explicitly list them for now?

Reply:
yeah there is a general clash that we do not disable transitive dependencies for test fixtures but that often clashes with the test class path then resulting  in conflicts etc.